modules/virt-creating-nad-localnet-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-nad-localnet-console_{context}"]
+= Creating a NAD for localnet topology using the web console
+
+You can create a network attachment definition (NAD) to connect workloads to a physical network by using the {product-title} web console.
+
+.Prerequisites
+* Use nmstate to configure a localnet identifier.
```

Comment:
should we link here to the openshift documentation where we explain how to configure this ? 

I mean ... Coming here without knowing what this is would leave me pretty confused - what's a localnet identifier ? 

Furthermore, maybe calling it "localnet to bridge mapping" would be more aligned with the rest of the documentation.

https://docs.openshift.com/container-platform/4.14/networking/multiple_networks/configuring-additional-network.html#configuring-additional-network_configuration-additional-network-interface

Reply:
@maiqueb IIUC, this configuration will be in the CLI version of these same docs? In which case, it will be included in this same assembly. I can't link directly to it because we can't cross-reference between modules (only assemblies). But if it's in the CLI docs, it will be nearby. 

If it's not in those docs, let me know and I'll figure something else out.

Reply:
> @maiqueb IIUC, this configuration will be in the CLI version of these same docs? In which case, it will be included in this same assembly. I can't link directly to it because we can't cross-reference between modules (only assemblies). But if it's in the CLI docs, it will be nearby.
> 
> If it's not in those docs, let me know and I'll figure something else out.

I don't follow - some of it is because I don't know the difference between an assembly and a module (I don't even know what those are ...).

I'm just asking to link to the openshift networking documentation where we show - using the CLI - how the ovn localnet bridge mappings are configured (which is a prerequisite of the UI you're developing).

Reply:
Sorry about the docs terminology! Done and done.